pkgs/by-name/cr/cryptpad/package.nix

```diff
@@ -59,6 +61,17 @@ let
     }
   ];
 
+  x2t_version = "v7.3+1";
+  x2t = fetchurl {
+    url = "https://github.com/cryptpad/onlyoffice-x2t-wasm/releases/download/${x2t_version}/x2t.zip";
+    hash = "sha256-hrbxrI8RC1pBatGZ76TAiVfUbZid7+eRuXk6lmz7OgQ=";
+  };
+  x2t_install = ''
+    local X2T_DIR=$out_cryptpad/www/common/onlyoffice/dist/x2t
+    ${lib.getExe unzip} ${x2t} -d "$X2T_DIR"
```

Comment:
(I'm never sure how these dependencies work with cross builds so I'd add unzip to nativeBuildInputs and call "unzip" directly here, but if you think this'll work I don't mind either way)

Reply:
Good point. I think this is correct: this is a build step that runs on the build machine, not the target architecture. That said, I see we're already specifyign other stuff (like `rdfind`) in `nativeBuildInputs`, so I'll change this to match.